src/components/Search/SearchBar/index.tsx

```diff
@@ -154,19 +152,20 @@ const SearchBar: React.FC<SearchBarProps> = ({
               <Icon glyph="Plus" />
             </IconButton>
           ) : (
-            <IconWithTooltip
-              data-cy="searchbar-warning"
-              fill={yellow.base}
-              glyph="Warning"
+            <Tooltip
+              justify="middle"
+              trigger={<IconPlaceholder data-cy="searchbar-error" />}
+              triggerEvent="hover"
             >
               {validatorMessage}
-            </IconWithTooltip>
+            </Tooltip>
           )
         }
         onChange={(e) => handleOnChange(e.target.value)}
         onKeyDown={handleKeyDown}
         placeholder="optional, regexp to search"
         spellCheck={false}
+        state={isValid ? "none" : "error"}
```

Comment:
I wonder if we can over ride this focus state and make it red too
<img width="690" alt="image" src="https://user-images.githubusercontent.com/4605522/230433611-516d1853-25bc-459d-82f8-1d4636041e17.png">